src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import AnimeCard from './Components/AnimePage';
+import Navbar from './Components/Navbar';
+import Details from './Components/AnimeDetails';
+
+function App() {
+  return (
+    <Router basename="/react-metrics-capstone">
+      <div className="App">
+        <Navbar />
+        <div>
+          <Routes>
+            <Route exact path="/" element={<AnimeCard />} />
+            <Route path="/Details" element={<Details />} />
```

Comment:
- [x] According to the requirements:

![image](https://github.com/Wils-FOURR-speed/react-metrics-capstone/assets/77069011/3ccd3a22-472b-414f-8432-14d16d7465cd)

Please consider using a dynamic route for each anime item details, at the moment, all anime details are in the same `/Details` route. Here's how you could achieve this easily:


```suggestion
            <Route path="/details/:id" element={<Details />} />
```
Then, in your anime card UI jsx, you could modify the route of the `Link`:
```js
// AnimePage.js
...
<Link
  to={`details/${anime.id}`}
  className="absolute top-2 left-38 p-2 font-bold bg-white text-sky-600 rounded-full hover:bg-sky-600 hover:text-white"
>
  <FaArrowRightLong onClick={() => dispatch(fetchAnimeDetailsById(anime.id))} className="h-6 w-6" />
</Link>

...

```

That's pretty much it! Now you have independent routes for each Anime item based on their id! 🤓 🔧